### PR TITLE
Fix hello_word_component.cpp :lines

### PR DESCRIPTION
**Pull request by gonidelis**
Documentation on 'How to build HPX components with pkg-config' hello_world_component.cpp was quoted incompletely 
`:lines7-29.` The correct is 
`:lines 7-30`

### CODE REVIEW

**Comment by gonidelis** [comment]
@msimberg Oh ok great! I shall fix all the `literalinclude`s then if that's not a problem as copyright and license commented lines will be removed from the manual html page.

File: docs/sphinx/manual/creating_hpx_projects.rst [hunk 1]
```diff
@@ -419,7 +425,9 @@ run it on.
 
 .. literalinclude:: ../../examples/quickstart/hello_world_1.cpp
    :language: c++
-
+   :start-after://[hello_world_1_getting_started
+   :end-before://]
```
Comment (msimberg) [inline docs/sphinx/manual/creating_hpx_projects.rst]:
```suggestion
   :start-after: //[hello_world_1_getting_started
   :end-before: //]
```

Comment (msimberg) [reply docs/sphinx/manual/creating_hpx_projects.rst]: And the same below, just for consistency with how we have it elsewhere.

Comment (gonidelis) [reply docs/sphinx/manual/creating_hpx_projects.rst]: Got it!

**Pull request merged**